boards/common/cdc_acm_serial.dts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+		zephyr,console = &cdc_acm_uart;
+		zephyr,shell-uart = &cdc_acm_uart;
+		zephyr,uart-mcumgr = &cdc_acm_uart;
+		zephyr,bt-mon-uart = &cdc_acm_uart;
+		zephyr,bt-c2h-uart = &cdc_acm_uart;
+	};
+};
+
+zephyr_udc0: &usbd {
+	cdc_acm_uart: cdc_acm_uart {
```

Comment:
should this not be `cdc_acm_uart0` if it supports multiple instances?

Reply:
`cdc_acm_uart0` is likely to conflict with samples or user applications, we should rather name it `board_cdc_acm_uart`, like we prefix snippets and shields.

Reply:
`board_*` prefix sounds good

Reply:
I also moved them to `boards/common/usb/`